brave/browser-laptop
# Adds broken seed overlay

Resolves #13424

## Submitter Checklist:


## Test Plan:

1) clean profile
2) enable payments
3) close browser
4) open `ledger-state.json` and modify `seed` (remove some of them)
5) start browser again
6) you should see an overlay
7) clicking on recover should open recover dialog

Expected overlay:
![image](https://user-images.githubusercontent.com/9574457/37800698-f35a2080-2de0-11e8-9ba4-0c208655090f.png)


## Reviewer Checklist:


Tests
